src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -77,7 +77,8 @@ export class DownloadCLIClientComponent extends Base implements OnInit {
     this.textDataRequirements = `
 ### Setup Command Line Interface
 ------------------------------
-Setup our Dockstore CLI application to start launching workflows from the command line.
+Setup our Dockstore CLI application in order to test workflows from the command line for [local development](https://docs.dockstore.org/en/stable/launch-with/launch.html#dockstore-cli), [validate .dockstore.yml files](https://docs.dockstore.org/en/stable/advanced-topics/dockstore-cli/yaml-command-line-validator-tool.html) for registering tools and workflows,
+run scripts or interact programmatically against Dockstore APIs, and [run workflows via the GA4GH WES standard](https://docs.dockstore.org/en/stable/advanced-topics/wes/wes-agc-tutorial.html) in platforms such as AWS AGC.
```

Comment:
Suggest spelling out Amazon Genomics Client instead of AGC -- I suspect most people won't know what it is.